### ISSUE TITLE
Feature Request: Execute only selected query

### ISSUE THREAD

**Issue by georgziganek** [hankinsoft/SQLPro#501]
When working on a file, I often often have multiple queries in the editor separated by a semicolon “;”.

It would be great to be able to define which queries I want to run:
- [ ] Command 1: Run active query (where the mouse courser is currently set)
- [ ] Command 2: Run all queries 
- [ ] have a shortcut for each of these commands

**Comment by hankinsoft** [comment]
Is this what you are looking for? https://www.sqlprostudio.com/blog/12-execute-query-in-cursor-range?

**Comment by georgziganek** [comment]
Yes exactly but I forgot to mention that I would like this for the iPadOS version.

**Comment by hankinsoft** [comment]
Sure, I'll take a look at getting this added. No ETA as of yet.

**Comment by georgziganek** [comment]
Great, thanks for the quick feedback!

**Comment by hankinsoft** [comment]
Going to mark this as closed. If you continue to see any issues, please let me know.
